pkg/unshare/unshare_linux.go

```diff
@@ -76,6 +78,26 @@ func getRootlessGID() int {
 	return os.Getegid()
 }
 
+func isSetID(path string, modeid fs.FileMode, capid capability.Cap) (bool, error) {
```

Comment:
It's okay to call `modeid` a `os.FileMode`.

Reply:
Now you just want to aggravate me....

Reply:
Trying to get you past [the linter](https://cirrus-ci.com/task/6722549160607744?logs=test#L16), which seems to not understand that package name.